test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/DaliDemo/DaliTableView.cs

```diff
@@ -133,14 +133,16 @@ public void Initialize()
             logo.ParentOrigin = new Position(0.5f, 0.1f, 0.5f);
             logo.WidthResizePolicy = ResizePolicyType.UseNaturalSize;
             logo.HeightResizePolicy = ResizePolicyType.UseNaturalSize;
+            float logoScale = (float)(NUIApplication.GetDefaultWindow().Size.Height) / 1080.0f;
```

Comment:
This change is not related to Popup.

Reply:
yes, absolutely right. but if this is not changed, all texts in sample app are not shown. (they shows as points => ... )
please accept this change. thank you.